### ISSUE TITLE
Redo front page to support multiple upcoming events

### ISSUE THREAD

**Issue by shrayasr** [Chennaipy/website#19]
Right now only 1 upcoming event can be shown

**Comment by astronomersiva** [comment]
If the scroll is really a concern, I also agree that a new design is needed. A change in the layout might  be sufficient. This opens up a lot of free space in the landing page. It might not look very good if that space is not utilized though.

![front](https://cloud.githubusercontent.com/assets/7725225/5629225/3c3ae938-95d7-11e4-9bd6-5fb3b87068ba.png)

**Comment by shrayasr** [comment]
This looks like the events are taking up a lot of space. In the case where we will have only one event (and I assume most of the months it'll be this way) then the whitespace would just become **too** much, no?

**Comment by astronomersiva** [comment]
That is precisely what I meant in the last line of the prev comment. But going by your assumption that we have not more than one event in a month, this issue of supporting multiple events shouldn't bother us much right?

**Comment by shrayasr** [comment]
Fair enough. I just left this here for record purposes, that when we need to support something like this, we can discuss it here.

Initially, I think it'll be enough to comma separate it. So if need be, we'll stick to that.

**Comment by bravegnu** [comment]
I was just thinking, if we should have a separate landing page, rendered outside pelican, may be written in HTML + pure CSS. I have created a separate issue for this #47.

**Comment by bravegnu** [comment]
Have a look at this, and let me know what you guys think. https://www.dropbox.com/sh/clhl27mzz6pzris/AABnP4t3POQJa1E0bIaogp3Aa?dl=0

**Comment by shrayasr** [comment]
> Have a look at this, and let me know what you guys think. https://www.dropbox.com/sh/clhl27mzz6pzris/AABnP4t3POQJa1E0bIaogp3Aa?dl=0

This looks killer! :+1: from me

**Comment by kskrishnasangeeth** [comment]
I have seen this template before at <code>purecss.io</code> when i was trying to learn <code>purecss</code> . This looks very good as a base template upon which we can build.  :+1:

Off topic but +1 for Hackathons too :smiley:

**Comment by bravegnu** [comment]
Yep, that was from the purecss landing page layout. http://purecss.io/layouts/marketing/

I should have posted this comment in #47  Will continue the discussion over there.
